analysis/show_question_dependencies.py

```diff
@@ -1,7 +1,7 @@
 import spacy
 import sys
 
-nlp = spacy.load('en')
+nlp = spacy.load('en_core_web_md')
```

Comment:
Heh, here I was commenting on Tom's work how we should probably change it back to `en` (in a different file) and here you change it from `en` to `en_core_web_md`. So I'll say that same thing here. I think we should move back to `en` and place that in the README as that is the "official" way on the LWP environment apparently. That does mean you have to enter another command to link them yeah. ;)